edu/jalc/inclass/cocacola/employee/EmployeeTest.java

```diff
@@ -1,35 +1,49 @@
 package edu.jalc.inclass.cocacola.employee;
 
+import edu.jalc.inclass.cocacola.security.PinNumber;
+
 public class EmployeeTest {
 
 	public void testNameOnlyConstructor(){
 		System.out.println("Testing Employee(String name)");
-		assert(false);
+		Employee employee = new Employee("Brandon");
+		assert(employee.getName() == "Brandon");
```

Comment:
You're only lucky this works. You need to compare `String`s with `.equals`. What is happening to allow it to work is the fact that the String literal, *Brandon* happens to be in the same address space throughout the execution of the code. The compiler/jvm is being smart enough to "catch" your mistake. It is seeing that "Brandon" is used multiple times and instead of wasting space with two instances of the same literal, it is keeping one until done. So this is technically broken. If the compiler/jvm didn't know the literal was coming (eg from user input, another computer ,etc) ,this allows for it to still be "correct" but possibly fail. I know, its way technical, so just go back and change all your string comparisons to use `.equals`. I will comment on each. When the PR shows everything outdated (eg you just need to push to your master and not worry about another PR), I will give you the 5 points.